pingo/compat.py

```diff
@@ -0,0 +1,28 @@
+import sys
```

Comment:
`compat.py` is a quite standard name to provide imports and functions that work both with Python 2 and Python 3.
Try https://www.google.fr/webhp?q=compat.py#safe=off&q=compat.py

https://docs.python.org/3/howto/pyporting.html